gruntfile.js

```diff
@@ -242,10 +242,10 @@ module.exports = function(grunt) {
   grunt.registerTask('save_deploy_info',
     function () {
       var options = this.options({
-        file: 'revision.properties'
+        file: 'Digiroad2Properties.revision'
       });
 
-      var data = ('digiroad2.latestDeploy=' + grunt.template.today('dd-mm-yyyy HH:MM:ss'));
+      var data = ('Digiroad2Properties.latestDeploy=' + grunt.template.today('dd-mm-yyyy HH:MM:ss'));
```

Comment:
Mitä funktionta nää toteuttaa enään? Varmaan kannattaa vaan poistaa. Toihan jopa kirjoittaa johonkin tiedostoon. Jos tätä samaa tietoa halutaan jonnekin, niin pitäis miettiä joku toinen paikka sille.

Reply:
siitä en nyt tiedä mut toi digiroad2.latestDeploy ja Digiroad2Properties.revision hommat vaikuttaa turhalta AWS maailmassa. Ajatus varmaan tuolla palvelimissa oli pitää lokia mitä versioita on asennettu ja milloin.

Reply:
Siinä tapauksessa tätä koko grunt.registerTaskia ei tarvita eli varmaan voin poistaa vaan nää: grunt.registerTask('save_deploy_info',
    function () {
      var options = this.options({
        file: 'Digiroad2Properties.revision'
      });
      grunt.file.write(options.file);
    }
  );

Reply:
Ok